backend/application/syson-sysml-import/src/main/java/org/eclipse/syson/sysml/parser/AstTreeParser.java

```diff
@@ -97,36 +99,50 @@ public List<EObject> parseJsonObject(final JsonNode astJson) {
         return result;
     }
 
-    public void resolveAllImport(final Resource rootResource) {
```

Comment:
When you change the APIs please add those breaking changes in the changelog/release notes.
Please verify each breaking change in this PR.